Annotation/polar_golden_set.json

```diff
@@ -1535,13 +1668,25 @@
             {
                 "predicate": [
                     {
-                        "str": "care",
+                        "str": "couldnt care",

```

Comment:
I wouldn't put the auxiliary in there

Reply:
yeh its a toughy - I've deleted the the aux, I think we could argue its only the second verb in the sentence as we could provide an ellision analysis, so the structure could be: it doesn't care [whether you have ...] and couldn't care [whether you have ...], with the first EC deleted and then there's a binding relation or some sort of resolution in LF - a potential idea

Reply:
Good point about the ellipsis, I think that's a good enough reason to do it like that